docs/api-reference/android/initialize.mdx

```diff
@@ -132,73 +42,93 @@ The Web3Auth Constructor takes an object with `Web3AuthOptions` as input. Constr
 | sdkUrl      | String                                  | No        | default is `"https://sdk.openlogin.com"`                                                                                            |
```

Comment:
they can't really pass any custom value to this param, so i think we should include it in docs.

Reply:
remove this right?

Reply:
removed